webapi/src/main/scala/org/knora/webapi/messages/util/search/gravsearch/transformers/OntologyInferencer.scala

```diff
@@ -129,6 +100,7 @@ final case class OntologyInferencer(private val ontologyCache: OntologyCache)(im
     statementPattern: StatementPattern,
     simulateInference: Boolean,
     limitInferenceToOntologies: Option[Set[SmartIri]] = None,
+    queryVariableSuffix: Option[String] = None,
```

Comment:
am I understanding correctly, that this parameter essentially only exists for testing?

Reply:
-ish. It would make sense to have even in the absence of testing, but as it stands, yes.